### DISCUSSION TITLE
[Feature Request] Allowing especial character display for Font Awesome

### DISCUSSION THREAD

**Discussion by LeCuay** [EddieDover/fvtt-party-sheet#31] [Ideas]
# Summary

This can get really in the middle of the parsing so I don't think embedding into the `text` property is a good idea, but maybe something like `Dialog` constructor, with `icon` property

**Comment by EddieDover** [comment]
Can you clarify what you're to achieve? A way to embed special characters from FA inside text elements?

**Reply by LeCuay** [reply]
Sure, what I'm thinking of are icons using `i` tag with classes to be displayed before the actual text

When the table grows big it's kinda hard to know which numbers are from which column (might also be my astigmatism as well), so I thought icons could be a good way to get what the numbers represent right away

**Reply by EddieDover** [reply]
So something like...
```json
      {
        "name": "STR Mod",
        "type": "direct",
        "header": "hide",
        "showSign": true,
        "text": "{fa fa-solid fa-hand-fist} system.abilities.str.mod"
      },
```
and seeing
![image](https://github.com/user-attachments/assets/e28c9b27-7f60-4efd-b440-54ccd2048f74)

**Reply by LeCuay** [reply]
Yeah that seems awesome! I'm assuming this is a local testing

**Reply by EddieDover** [reply]
It is, but I'll be happy to generate a dev build for you to test

**Reply by EddieDover** [reply]
[fvtt-party-sheet.zip](https://github.com/user-attachments/files/16662115/fvtt-party-sheet.zip)

**Comment by EddieDover** [comment]
Closing this discussion since I think we've wrapped up the functionality sans testing.  Feel free to check out #33 and test if you want to.
